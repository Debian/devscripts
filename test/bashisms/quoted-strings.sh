#!/bin/sh

foo="
echo -e nothing wrong here
#crap"

echo -e BASHISM

foo="\
#crap"

echo -e BASHISM

case foo in
    *\'*)
	echo -e BASHISM
	;;
esac
#'
echo -e BASHISM

case foo in
    *\\"*")
	echo -e BASHISM
	;;
    *\\\"*)
	echo -e BASHISM
	;;
    *\"*)
	echo -e BASHISM
	;;
esac
#"
echo -e BASHISM

foo='\'
echo -e BASHISM
