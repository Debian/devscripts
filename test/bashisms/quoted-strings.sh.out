possible bashism in bashisms/quoted-strings.sh line 7 (echo -e):
echo -e BASHISM
possible bashism in bashisms/quoted-strings.sh line 12 (echo -e):
echo -e BASHISM
possible bashism in bashisms/quoted-strings.sh line 16 (echo -e):
	echo -e BASHISM
possible bashism in bashisms/quoted-strings.sh line 20 (echo -e):
echo -e BASHISM
possible bashism in bashisms/quoted-strings.sh line 24 (echo -e):
	echo -e BASHISM
possible bashism in bashisms/quoted-strings.sh line 27 (echo -e):
	echo -e BASHISM
possible bashism in bashisms/quoted-strings.sh line 30 (echo -e):
	echo -e BASHISM
possible bashism in bashisms/quoted-strings.sh line 34 (echo -e):
echo -e BASHISM
possible bashism in bashisms/quoted-strings.sh line 37 (echo -e):
echo -e BASHISM
